src/main/java/org/opensearch/performanceanalyzer/decisionmaker/deciders/Decision.java

```diff
@@ -57,7 +57,8 @@ public boolean isEmpty() {
 
     @Override
     public FlowUnitMessage buildFlowUnitMessage(String graphNode, InstanceDetails.Id node) {
-        // All deciders run on the master node, (in initial versions), so we dont expect Decisions
+        // All deciders run on the cluster_manager node, (in initial versions), so we don't expect
+        // Decisions
```

Comment:
nit : can be a single line
```suggestion
        // All deciders run on the cluster_manager node, (in initial versions), so we don't expect Decisions
```

Reply:
This change is amended in the last commit.